We need the xptcstubsdef_asm file for Sun Studio, but not necessary for GCC. 
One of the files was placed in the wrong location and needs to be moved. This 
part can probably be sent upstream. 

diff --git a/xpcom/reflect/xptcall/md/unix/moz.build b/xpcom/reflect/xptcall/md/unix/moz.build
--- a/xpcom/reflect/xptcall/md/unix/moz.build
+++ b/xpcom/reflect/xptcall/md/unix/moz.build
@@ -267,9 +270,6 @@
     ]
 
 if CONFIG['OS_ARCH'] == 'SunOS' and '86' in CONFIG['OS_TEST']:
-    GENERATED_FILES = [
-        'xptcstubsdef_asm.solx86',
-    ]
     if CONFIG['OS_TEST'] == 'x86_64':
         if CONFIG['GNU_CC']:
             SOURCES += [
